cli/command/container/utils.go

```diff
@@ -92,6 +94,7 @@ func waitExitOrRemoved(ctx context.Context, dockerCli *command.DockerCli, contai
 			}
 		}
 	}()
+	<-started
```

Comment:
It just makes sure that at least the goroutine is running before it returns... which then will make a call to start the container.